log/logger.ts

```diff
@@ -22,7 +24,8 @@ export class Logger {
     this.handlers = handlers || [];
   }
 
-  _log(level: number, msg: string, ...args: any[]) {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  _log(level: number, msg: string, ...args: any[]): void {
```

Comment:
Could this be `unknown[]` instead so type safety isn’t lost?

Reply:
Possibly, something we can try to clean up later.  There is quite a bit of stuff that doesn't work under `strict` mode either... Ry and I agreed to stop trying to land all the cleanup in this and just get it running.